internal/app/machined/pkg/controllers/network/ethernet_status.go

```diff
@@ -135,6 +135,11 @@ func (ctrl *EthernetStatusController) reconcile(
 			lgger.Warn("error getting rings", zap.Error(err))
 		}
 
+		features, err := ethClient.Features(iface)
```

Comment:
How does this part know when it's (time of update + some time for it to be readable) to re-read this?